packages/react-native-otlp/RNEmbraceOTLP.podspec

```diff
@@ -1,22 +1,20 @@
 require "json"
+
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
 
 Pod::Spec.new do |s|
-  s.name = "RNEmbraceOTLP"
-  s.version = package["version"]
-  s.summary = package["description"]
-  s.homepage = package["homepage"]
-  s.license = package["license"]
-  s.authors = package["author"]
-
-  s.ios.deployment_target = '13.0'
```

Comment:
it was, because the `create-react-native-library` did not have this min version added. but I think since it was already tested in the iOS build I would re-introduce it

Reply:
ya I think this would be more specific to our minimum requirements dictated by the iOS SDK. Actually I think we'd be fine leaving these off and only including them in the core package's podspec? Maybe a question for the iOS team to confirm but since they are specific to the SDK requirements that is the only package that references it directly so maybe not needed here after all. If we do need them in multiple packages we'll probably want to read from package.json to keep consistent

Reply:
yes, the core package already contains the following

```
  s.ios.deployment_target = '13.0'
  s.tvos.deployment_target = '13.0'
  s.swift_version = '5.0'
```

ok, so I'll remove the `s.ios.deployment_target = '13.0'` from the OTLP package since this is behaving as a direct dependency of Core